elementary/monitor/fetchers/tests/schema.py

```diff
@@ -1,11 +1,17 @@
-from typing import List, Optional, Union
+from typing import List, Literal, Optional, Union
 
 from pydantic import Field, validator
 
+from elementary.monitor.api.models.schema import NormalizedArtifactSchema
 from elementary.utils.schema import ExtendedBaseModel
 from elementary.utils.time import convert_partial_iso_format_to_full_iso_format
 
 
+class NormalizedTestSchema(NormalizedArtifactSchema):
```

Comment:
I dislike how this is here while all the others (model, source, exposure) are at `fetchers/models/schema.py` even though they don't store models.